framework/src/modules/chain/modules/transactions.js

```diff
@@ -253,6 +250,17 @@ Transactions.prototype.transactionInPool = function(id) {
 	return __private.transactionPool.transactionInPool(id);
 };
 
+/**
+ * Fills pool.
+ *
+ * @param {function} cb - Callback function
+ * @returns {function} Calls transactionPool.fillPool
+ * @todo Add description for the params
+ */
+Transactions.prototype.fillPool = function(cb) {
+	return __private.transactionPool.fillPool(cb);
```

Comment:
probably transaction pool shouldn't be referenced in `transactions` module.
Maybe we can fix in some other issue?

Reply:
Why do you think it shouldn't be referenced here?

Reply:
nvm, i think the direction of dependency is correct 👍